reverse-proxy-traefik/readme.md

```diff
@@ -0,0 +1,142 @@
+# Reverse Proxy Traefik
+
+The package is an alternative reverse proxy nginx, this reverse proxy exposes packages using both domain and subdirectory to host the following services:
+
+| Package  | Hosted                                    |
+|----------|-------------------------------------------|
+| Superset | Domain                                    |
+| Jempi    | Domain                                    |
+| Santempi | Domain                                    |
+| Kibana   | Domain                             |
+| Minio    | Sub Directory                             |
+| Grafana  | Sub Directory                             |
+| JSReport | Sub Directory                             |
```

Comment:
We can have this share a subdomain e.g `monitoring.<domain>/minio `... Also amend the respective docs.